docs/hyperopt.md

```diff
@@ -471,6 +471,7 @@ Currently, the following loss functions are builtin:
 * `SortinoHyperOptLossDaily` - optimizes Sortino Ratio calculated on **daily** trade returns relative to **downside** standard deviation.
 * `MaxDrawDownHyperOptLoss` - Optimizes Maximum absolute drawdown.
 * `MaxDrawDownRelativeHyperOptLoss` -  Optimizes both maximum absolute drawdown while also adjusting for maximum relative drawdown.
+* `MaxDrawDownPerPairHyperOptLoss` -  Optimizes the profit/drawdown ratio calculated individually for each pair in the pairlist.
```

Comment:
I think this doesn't really explain the point of this well.
the docstring of the function itself is better, really